sw-emulator/lib/periph/src/doe.rs

```diff
@@ -410,4 +413,26 @@ mod tests {
         assert_eq!(soc_reg.doe_key(), expected_doe_key);
         assert_eq!(soc_reg.field_entropy(), expected_fe);
     }
+
+    #[test]
+    fn test_cbc() {
```

Comment:
How is this related to cert testing?

Reply:
Oh, I just realized this is a PR into your own branch. Never mind :)

Reply:
I've since rebased onto main (since that other PR has been submitted).